tests/default_vars.sh

```diff
@@ -449,3 +453,106 @@ export_35d_run ()
 export CNTL_DIR=""
 export LIST_FILES=""
 }
+export_datm ()
+{
+export FV3=false
+export S2S=false
+export DATM=true
+export DAYS=1
+export FHMAX=24
+export WLCLK=30
+export THRD=1
+export FHROT='0'
+
+# atm/ocn/ice resolution
+# GEFS
+export DATM_SRC="GEFS"
+export FILENAME_BASE='gefs.'
+export IATM=1536
+export JATM=768
+export ATMRES='C96'
+export OCNRES='100'
+export ICERES='1.00'
+export NX_GLB=360
+export NY_GLB=320
+
+# nems.configure
+export MED_restart_data=''
+export MEDCOMP="cmeps"
+export NEMS_CONFIGURE="nems.configure.datm.IN"
+export med_model="nems"
+export atm_model="datm"
+export ocn_model="mom6"
+export ice_model="cice6"
+export atm_petlist_bounds="0 15"
+export med_petlist_bounds="16 111"
+export ocn_petlist_bounds="112 231"
```

Comment:
This is giving 120 PEs to the ocean at 1deg resolution and 48 PEs to the ice. We should set appropriate values for the default 1deg resolution and then change them in the 1/4deg test.

Reply:
Will we have an issue and future PR to correctly assign the right PEs for 1 deg ocean and ice?

Reply:
The model runs very quickly using these PEs for the 1 degree ocean and ice.

Reply:
I'm sure it does. But does it really seem reasonable to assign the same PES whether you're running 1/4 deg or 1deg?

Reply:
How many PEs are 1 degree MOM6 and CICE6 using in the coupled model? We do want to use fewer tasks for low resolution runs. Can we include this in the next DATM PR?

Reply:
The 1/4 deg set up in the coupled model is OCN on 30 PE and ICE on 12. When I set up the coupled model tasking, I simply reduced the 1/4 deg by 50% for the 1/2 deg and another 50% for the 1deg. So 120,60,30 for the ocean and 48,24,12 for the ice.

Phil is running even lower PE counts I believe.